tests/http/test_http_parser.py

```diff
@@ -17,18 +17,22 @@
 from proxy.http import httpStatusCodes, httpMethods
 from proxy.http.parser import HttpParser, httpParserTypes, httpParserStates
 
```

Comment:
No need of an extra line.  Idea is to keep all sub-module level imports together.

Reply:
Agree, Will remove the extra line